catt/controllers.py

```diff
@@ -26,6 +26,20 @@ def get_stream_info(video_url):
     return stream_info
 
 
+def get_chromecast(device_name):
+    devices = pychromecast.get_chromecasts()
+    if device_name:
```

Comment:
That looks nice, I just think that for testing purposes, it is useful to be able to distinguish between "specified not found" and "none found". But we can do it like you suggest if you prefer.